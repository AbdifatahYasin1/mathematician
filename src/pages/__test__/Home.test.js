// import React from 'react';
// import { render } from '@testing-library/react';
// import renderer from 'react-test-renderer';
// import Home from '../Home';

// describe('Testing Home', () => {
//   test('should render correctly', () => {
//     render(<Home />);
//   });
// });

// test('should have a length of 1', () => {
//   const { getAllByText } = render(<Home />);
//   const text = getAllByText(/What is Mathematics?/);
//   expect(text).toHaveLength(1);
// });

// it('fisrt snap shot test', () => {
//   const tree = renderer.create(<Home />).toJSON();
//   expect(tree).toMatchSnapshot();
// });
