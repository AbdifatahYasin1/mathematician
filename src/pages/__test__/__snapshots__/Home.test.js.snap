// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fisrt snap shot test 1`] = `
<div
  id="home"
>
  <h1>
    What is Mathematics?
  </h1>
  <p>
    Mathematics is the study of numbers, shapes, and patterns. 
  </p>
</div>
`;
